### ISSUE TITLE
Can I use arduino pro micro with zoom g1xon via usb without host shield?

### ISSUE THREAD

**Issue by e36910** [tttapa/Control-Surface#850]
Wich library will be to connect for working zoom devices?
![IMG_20221117_233924_Mi9T_g8 4 LMC r9 v1](https://user-images.githubusercontent.com/85799952/202865482-d3e767f2-71fa-4235-8c98-3ee692cdcf78.jpg)

**Comment by tttapa** [comment]
You cannot connect two USB devices to each other, you need at least one host. The ATmega32U4 only supports device mode.

The Zoom's manual also doesn't mention MIDI at all, it seems that the USB connection only supports firmware upgrades, are you sure it supports MIDI over USB?

**Comment by e36910** [comment]
Yes, I`m sure it supports MIDI over USB. Look this video - [https://youtu.be/rHc8-jrd__c?t=220](url)

**Comment by tttapa** [comment]
Okay, that's good. Either way, you'll need a USB host (unless the Zoom supports USB OTG, which I doubt).

**Comment by e36910** [comment]
How do I check the zoom for USB OTG support?

**Comment by tttapa** [comment]
You'll have to check the manual/documentation or contact the manufacturer. Or you can just try an OTG adapter. But I'm 99% sure it doesn't support OTG, otherwise they would advertise it.

**Comment by e36910** [comment]
How to connect USB host shield with arduino pro micro?

**Comment by tttapa** [comment]
I'm afraid that's not something I can help you with. I'm sure you can find more information online, but if you need more help after reading the online instructions, you could open a thread on https://forum.arduino.cc/.
